tests/integration/test_ols.py

```diff
@@ -472,12 +473,14 @@ def validate_question(conversation_id, query):
     assert query_passed == "test query"
 
 
+@pytest.mark.attachment
```

Comment:
`@pytest.mark.attachment()`?

Reply:
indeed, let's be consistent, you are right

Reply:
fixed @onmete